### PR TITLE
doc/add voltbuilder to tools

### PR DESCRIPTION
**Pull request by gwhenne**
### Platforms affected

documentation only.

### Motivation and Context
Add a new tool to the list of tools using Cordova

### Description
Add a short description and an icon

### Testing
Loaded the existing page, applied the changes and examined the result.

### Checklist

- [x] I've run the tests to see all new and existing tests pass
- [x] I added automated test coverage as appropriate for this change
- [x] Commit is prefixed with `(platform)` if this change only applies to one platform (e.g. `(android)`)
- [x] If this Pull Request resolves an issue, I linked to the issue in the text above (and used the correct [keyword to close issues using keywords](https://help.github.com/articles/closing-issues-using-keywords/))
- [x] I've updated the documentation if necessary

### CODE REVIEW

**Review by NiklasMerz — APPROVED** [review]

**Review by timbru31 — APPROVED** [review]
LGTM

File: www/_data/tools.yml
Comment (jcesarmobile) [inline www/_data/tools.yml:107]:
```suggestion
        Modern replacement for PhoneGap Build. Upload project and certificates,
```

Phonegap is not the same as Phonegap Build, VoltBuilder is not a Phonegap replacement

**Review by jcesarmobile — CHANGES_REQUESTED** [review]
Added a comment and also sent a mail because there are some problems with their site, please, do not merge until it's addressed.

**Review by breautek — APPROVED** [review]
lgtm, and I think @jcesarmobile concerns were addressed, but I'm not sure which pages were offending.

I checked their [About](https://volt.build/about/) page and a few others and they all appear to reference PhoneGap Build instead of PhoneGap.

**Comment by gwhenne** [comment]
Sorry - this one fell behind my desk. All references on our site to Cordova should now read Apache Cordova.

**Comment by brodybits** [comment]
- I noticed "Replace PhoneGap." on the main page of https://volt.build/ and the link to a "migration guide" seems to be broken.
- I noticed there is a table with just "PhoneGap" in a column header.
- We had a bad experience in the past where PhoneGap Build would force people to use older versions of some Cordova platforms and would appreciate some reassurance that there would be flexibility in the future.
- I would also appreciate a comparison to https://ionicframework.com/appflow which seems to support building with both Cordova and Capacitor
- I would also advise resolving the conflict on GitHub

**Review by jcesarmobile — APPROVED** [review]
I think those changes are not of our business, as long as they use Apache cordova referring to us I don’t mind how they refer to others or their competitors.

The header is about phonegap because they created the project when phonegap announced they were shutting down. There is a row where it specify the Apache cordova version used, 8.0.0 for phonegap, 10.0.0 for voltbuilder.

I approve but the conflict needs to get fixed before we can merge.

**Comment by gwhenne** [comment]
Thanks for the constructive comments!
- Link on main page fixed - thanks!
- Column header now says "PhoneGap Build"
- We suffered as PhoneGap users too - it was what drove us to make VoltBuilder. We added a statement to the About page stating we continuously update the VoltBuilder toolchain to the latest versions.
- We're not qualified to do a comparison to AppFlow - but we'll be happy to work with anyone interested in doing one.
- The conflict on GitHub has been resolved.

**Comment by breautek** [comment]
Checks are green. Several Green checks, no explicit red checks. Merging!

Thank you @gwhenne for taking the time for this PR and applying the requested changes. Hope VoltBuilder does well.

**Pull request merged**
